pkg/services/rpcsrv/subscription_test.go

```diff
@@ -193,6 +194,20 @@ func TestSubscriptions(t *testing.T) {
 		}
 	}
 
+	signer, _ := util.Uint160DecodeStringLE("89abcdef0123456789abcdef0123456789abcdef")
```

Comment:
Use one of testchain's keys for it, e.g.  `testchain.PrivateKeyByID(0).PublicKey().GetScriptHash()`. It should have some balance on the account.